web/app/js/components/BreadcrumbHeader.test.jsx

```diff
@@ -26,4 +26,45 @@ describe('Tests for <BreadcrumbHeader>', () => {
     const crumbs = component.find("span");
     expect(crumbs).toHaveLength(3);
   });
+
```

Comment:
Thanks for adding these tests! They help a lot with understanding. For completeness can we also add the single namespace case?

```suggestion
  it("renders correct breadcrumb text for top-level pages [Namespaces]", () => {

    loc.pathname = "/namespaces";
    const component = mount(
      <BrowserRouter>
        <BreadcrumbHeader
          location={loc}
          pathPrefix="" />
      </BrowserRouter>
    );

    const crumbs = component.find("span");
    expect(crumbs).toHaveLength(1);
    const crumbText = crumbs.reduce((acc, crumb) => {
      acc+= crumb.text();
      return acc;
    }, "")
    expect(crumbText).toEqual("Namespaces");
  });

  it("renders correct breadcrumb text for top-level pages [Control Plane]", () => {

    loc.pathname = "/controlplane";
    const component = mount(
      <BrowserRouter>
        <BreadcrumbHeader
          location={loc}
          pathPrefix="" />
      </BrowserRouter>
    );

    const crumbs = component.find("span");
    expect(crumbs).toHaveLength(1);
    const crumbText = crumbs.reduce((acc, crumb) => {
      acc+= crumb.text();
      return acc;
    }, "")
    expect(crumbText).toEqual("Control Plane");
  });
```

Reply:
Great idea 👏  will do!